src/c++/vdo/base/dm-vdo-target.c

```diff
@@ -1068,10 +1068,30 @@ static void vdo_io_hints(struct dm_target *ti, struct queue_limits *limits)
 	limits->logical_block_size = vdo->device_config->logical_block_size;
 	limits->physical_block_size = VDO_BLOCK_SIZE;
 
+#ifndef VDO_UPSTREAM
+#undef VDO_USE_ALTERNATE
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(10, 0))
+#define VDO_USE_ALTERNATE
+#endif
+#else /* !RHEL_RELEASE_CODE */
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 10, 0))
```

Comment:
Same comment here as for dmTracer.c. I think this should specify kernel version 6.12 in the comparison.